05_DataQuality.R

```diff
@@ -645,10 +645,35 @@ dkr_destination <- base_dq_data %>%
 missing_destination_subsidy <- base_dq_data %>%
   filter(!is.na(ExitDate) &
            Destination == 435 &
-           is.na(DestinationSubsidyType)) %>%
+           (is.na(DestinationSubsidyType) |
+           !DestinationSubsidyType %in% c(subsidy_types))) %>%
   merge_check_info(checkIDs = 121) %>%
   select(all_of(vars_we_want))
 
+# Missing ResPrior Subsidy ------------------------------------------------
+
+missing_res_prior_subsidy <- base_dq_data %>%
```

Comment:
why "prior"?

Reply:
:) I think a long time ago, Enrollment.LivingSituation was called Enrollment.ResidencePrior? Either way, even now, that subset of data elements (see the screenshot) is called "Prior Living Situation." The meaning of LivingSituation in the Enrollment table is the place the household is coming from, or, their "residence prior" to entry into whatever project it is. As opposed to "CurrentLivingSituation." 
![image](https://github.com/abtassociates/eva/assets/7561522/2203d73c-4f2e-4866-b6cb-2aced33e10c5)